substratest/factory.py

```diff
@@ -142,6 +142,7 @@ class Permissions:
 
 
 DEFAULT_PERMISSIONS = Permissions(public=True, authorized_ids=[])
+DEFAULT_OUT_MODEL_PERMISSIONS = Permissions(public=False, authorized_ids=[])
```

Comment:
shouldn't this have at least one authorized node? as is not public?

Reply:
I believe passing an empty list will be interpreted as "owner only".

Reply:
yes you are right, the backend will add the current node permission